uber/models.py

```diff
@@ -856,6 +861,7 @@ def model_mixin(cls, model):
 
 
 class Group(MagModel, TakesPaymentMixin):
+    search_key    = Column(UUID, default=lambda: str(uuid4()))
```

Comment:
Done. Now it'll handle the search term intelligently even if the search term itself happens to be the EVENT_QR_ID.